cogs/currencycog.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import discord
+from discord.ext import commands
+
+
+class CurrencyCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def cconvert(self, ctx, *args):
```

Comment:
Since no complex parsing is being done on the arguments, we can have explicit arguments in the parameters.

E.g.
```python
@commands.command()
async def cconvert(self, ctx, base, target, amount):
```

This is preferable because `args[0]`, `args[1]`, etc. are not very descriptive as to what they contain, and this way there is no need to explicitly check for sufficient arguments passed!

Reply:
Furthermore, I believe you can avoid needing to explicitly convert `amount` to a float by specifying its type like so

```python
@commands.command()
async def cconvert(self, ctx, base, target, amount: float):
```